libs/execution_context/ebpf_maps.c

```diff
@@ -2671,6 +2685,12 @@ ebpf_map_delete_entry(_In_ ebpf_map_t* map, size_t key_size, _In_reads_(key_size
         return EBPF_OPERATION_NOT_SUPPORTED;
     }
 
+    if (flags & EBPF_MAP_FLAG_HELPER && map->ebpf_map_definition.key_size != 0) {
```

Comment:
```suggestion
    if ((flags & EBPF_MAP_FLAG_HELPER) && map->ebpf_map_definition.key_size != 0) {
```
for consistency with line 2619